src/phases/query/RepeatedPathTraversal.yml

```diff
@@ -20,305 +20,258 @@ GlobalDefaults:
-              Filter: {^Parameter: {Name: "Filter", Default: "invalid"}}
-
-Actors:
-# Clear any pre-existing collection state.
-- Name: ClearCollection
-  Type: CrudActor
-  Database: *Database
-  Phases:
-    OnlyActiveInPhases:
-      Active: [0]
-      NopInPhasesUpTo: *MaxPhases
-      PhaseConfig:
-        Repeat: 1
-        Threads: 1
-        Collection: *Collection
-        Operations:
-        - OperationName: drop
+            - OperationName: drop
 
-# Insert documents with 2 levels of nested arrays where the leaves contain many fields. This test
-# targets workloads where the paths queried share a common prefix ("a.b.c", "a.b.d", and so on).
-- Name: InsertRepeatedPathTraversalData
-  Type: Loader
-  Threads: 1
-  Phases:
-    OnlyActiveInPhases:
-      Active: [1]
-      NopInPhasesUpTo: *MaxPhases
-      PhaseConfig:
-        Repeat: 1
-        Database: *Database
-        Threads: 1
-        CollectionCount: 1
-        DocumentCount: *DocumentCount
-        BatchSize: 1000
-        Document: {
-          a: [{ ^Repeat: { count: 3, fromGenerator: { b: [&InnerDoc { c: 0, d: 0, e: 0, f: 0, g: 0, h: 0, i: 0, j: 0, k: 0, l: 0}]}}}],
-          # 5 levels deep
-          deep5: [
-            {f1: [
-              {f2: [
-                {f3: [
-                  {f4: [
-                    {f5: [*InnerDoc]},
-                    {f5: [*InnerDoc]},
-                  ]},
-                ]},
-              ]},
-            ]},
-          ],
-          # 10 levels deep
-          deep10: [
-            {f1: [
-              {f2: [
-                {f3: [
-                  {f4: [
-                    {f5: [
-                      {f6: [
-                        {f7: [
-                          {f8: [
-                            {f9: [
-                              {f10: [*InnerDoc]},
-                              {f10: [*InnerDoc]},
-                            ]},
-                          ]},
-                        ]},
-                      ]},
-                    ]},
-                  ]},
-                ]},
-              ]},
-            ]},
-          ],
-        }
+  # Insert documents with 2 levels of nested arrays where the leaves contain many fields. This test
+  # targets workloads where the paths queried share a common prefix ("a.b.c", "a.b.d", and so on).
+  - Name: InsertRepeatedPathTraversalData
+    Type: Loader
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [1]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Repeat: 1
+          Database: *Database
+          Threads: 1
+          CollectionCount: 1
+          DocumentCount: *DocumentCount
+          BatchSize: 1000
+          Document: {
+            a: [{^Repeat: {count: 3, fromGenerator: {b: [&InnerDoc {c: 0, d: 0, e: 0, f: 0, g: 0, h: 0, i: 0, j: 0, k: 0, l: 0}]}}}],
+            # 5 levels deep
+            deep5: [{f1: [{f2: [{f3: [{f4: [{f5: [*InnerDoc]}, {f5: [*InnerDoc]}]}]}]}]}],
```

Comment:
I find it a bit odd that it adds spaces for some `}}` combos but not for others. e.g [here](https://github.com/mongodb/genny/pull/1205/files?diff=split&w=1#diff-171b956f545f03dd1edd5321d1bcf1e9a081f51a4bc4298d2a6480f044da87a6R16) or [here](https://github.com/mongodb/genny/pull/1205/files?diff=split&w=1#diff-a9d48c8939e97aec7c441433a0b31da6cf17c0e3cdde684c181cd9214f5de6d0R45)